Extension/src/LanguageServer/Providers/codeActionProvider.ts

```diff
@@ -218,7 +226,47 @@ export class CodeActionProvider implements vscode.CodeActionProvider {
                 kind: codeActionKind
             };
             resultCodeActions.push(vscodeCodeAction);
-        });
+        };
+
+        response.commands.forEach(processCommand);
+
+        // If the refactor.inline.macro code action is specifically invoked by the user,
+        // then force a hover to ensure that the "Expands to" info is available.
+        if (!hasInlineMacro && context.only?.value === "refactor.inline.macro") {
+            const processInlineMacro = async (): Promise<boolean> => {
+                const editor: vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+                if (!editor) {
+                    return false;
+                }
+                const result: vscode.Hover[] = <vscode.Hover[]>(await vscode.commands.executeCommand('vscode.executeHoverProvider', document.uri, range.start));
+                if (result.length === 0) {
+                    return false;
+                }
+                const hoverResult: vscode.MarkdownString = <vscode.MarkdownString>result[0].contents[0];
+                if (!hoverResult.value.includes(localize("expands_to", "Expands to:"))) {
```

Comment:
nit: we usually use `.` instead of `_` for localization keys. (`expands.to`)

Reply:
ok